### ISSUE TITLE
User Survey in an issue: Anything we should change for 1.0?

### ISSUE THREAD

**Issue by bwplotka** [bwplotka/bingo#135]
Hi,

It's time of the tool stability in terms of flags and behaviour, that I would love to consider releasing v1.0 finally (: This means no flag and option can change without breaking change. In terms of .bingo directory structure, any changes would need to be also limited, however with migration scheme as I was doing we could change things. In v1.0 we can also add more stuff.

Please add +1 emoji if you don't have objections. If you have some wishes towards API, now is time to comment here!  🤗 

I will be waiting until EOM March, then release v1.0.

**Comment by kishaningithub** [comment]
Not sure if this the right place to put this, i would like to have a `bingo get -u` option similar to `go get -u` to upgrade all my dependencies using a single command

**Comment by bwplotka** [comment]
No, it's a good mention. However we can totally add this after 1.0 (: Added https://github.com/bwplotka/bingo/issues/139 for this.
